lib/upipe-modules/upipe_rtp_fec.c

```diff
@@ -506,7 +506,7 @@ static void upipe_rtp_fec_timer(struct upump *upump)
         if (upipe_rtp_fec->last_send_seqnum != UINT32_MAX) {
             uint16_t expected = upipe_rtp_fec->last_send_seqnum + 1;
             if (expected != seqnum) {
-                upipe_dbg_va(upipe, "FEC output LOST, expected seqnum %hu got %hu",
+                upipe_err_va(upipe, "FEC output LOST, expected seqnum %hu got %hu",
```

Comment:
A warning perhaps ? error = unrecoverable

Reply:
Changed to warning